resources/js/app/components/dialog/dialog.js

```diff
@@ -70,13 +72,21 @@ export const Dialog = Vue.extend({
             confirmCallback: this.hide,
             cancelMessage: false,
             inputValue: '',
-            checkValue: '',
-            checkLabel: '',
+            checks: [],
             loading: false,
             unique: [],
         };
     },
     methods: {
+        callbackOptions(title, unique) {
+            const relations = this.checks?.filter(c => c?.name === 'relations')?.[0]?.value || false;
+            const translations = this.checks?.filter(c => c?.name === 'translations')?.[0]?.value || false;
```

Comment:
You can use `.find` instead `.filter` so you don't have to get first element of array